src/python/WMCore/Services/LogDB/LogDB.py

```diff
@@ -6,86 +6,91 @@
 
 # standard modules
 import os
+import logging
 
 # project modules
 from WMCore.Services.LogDB.LogDBBackend import LogDBBackend
 
+def remove_last_slash(url):
+    "Remove last slash from url"
+    while True:
+        if  url.endswith('/'):
+            url = url[:-1]
+        else:
+            break
+    return url
+
+def url_db(url):
+    "Split given url into couchdb url and db name"
+    pos = remove_last_slash(url).rfind('/')
+    return remove_last_slash(url[:pos]), remove_last_slash(url[pos+1:])
+
```

Comment:
Maybe, we can use the following function since it is already there.
https://github.com/vkuznet/WMCore/blob/logdb2/src/python/WMCore/Lexicon.py#L554